### PR TITLE
Add Learn PWA: Part 22, Complexity.

### LINKED ISSUE #6501: Learn PWA: 23. Complexity management

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #6501

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

File: src/site/content/en/learn/pwa/complexity/index.md [hunk 1]
```diff
@@ -1,11 +1,126 @@
...
+  Keeping a web app simple can be surprisingly complicated. In this module, you will learn how web APIs work with threading and how you can use this for common PWA patterns such as state management.
+date: 2022-04-15
 ---
 
-Coming soon!
+## Simplicity and complexity
+
+In his talk [Simple Made Easy](https://www.infoq.com/presentations/Simple-Made-Easy/), Rich Hickey discusses the qualities of simple versus complex things. He describes simple things as focusing on:
+
+* One role, one task, one concept, or one dimension.
```
Comment (rachelandrew) [inline src/site/content/en/learn/pwa/complexity/index.md:14]:
```suggestion
> "One role, one task, one concept, or one dimension."
```

File: src/site/content/en/learn/pwa/complexity/index.md [hunk 2]
```diff
@@ -1,11 +1,126 @@
...
+
+But emphasizes that simplicity isn't about:
+
+* Having one instance or performing one operation.
```
Comment (rachelandrew) [inline src/site/content/en/learn/pwa/complexity/index.md:18]:
```suggestion
> "Having one instance or performing one operation."
```

Comment (rachelandrew) [reply src/site/content/en/learn/pwa/complexity/index.md:18]: these two have ended up as a single item list each, they seem to actually be quotes so maybe what I've done works, I don't think they should be list items though.

**Review by rachelandrew — APPROVED** [review]
@jpmedley see my note about the two items in the intro, I've approved so this isn't a blocker to merge though.

**Review by devnook — APPROVED** [review]
This is a test comment.

**Pull request merged**